config-vm.md

```diff
@@ -0,0 +1,37 @@
+# Virtual-machine-specific Configuration
+
+Virtual-machine-based runtimes require additional configuration to that specified in the [default spec configuration](config.md).
+
+This optional configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) path to file that represents the root filesystem for the virtual machine.
+* **`kernel`** (object, required) specifies details of the kernel to boot the virtual machine with.
+
+Note that `imagePath` refers to a path on the host (outside of the virtual machine).
+This field is distinct from the **`path`** field in the (#Root-Configuration) section since in the context of a virtual-machine-based runtime:
+
+* **`ImagePath`** will represent the root filesystem for the virtual machine.
+* The container root filesystem specified by **`path`** will be mounted inside the virtual machine at a location chosen by the virtual-machine-based runtime.
+
+The virtual-machine-based runtime will use these two path fields to arrange for **`path`** to be presented to the process to run as the root filesystem.
```

Comment:
On Fri, May 20, 2016 at 03:28:37PM -0700, Mrunal Patel wrote:
> > +* **`ImagePath`** will represent the root filesystem for the virtual machine.
> > +* The container root filesystem specified by **`path`** will be mounted inside the virtual machine at a location chosen by the virtual-machine-based runtime.
> > +
> > +The virtual-machine-based runtime will use these two path fields to arrange for **`path`** to be presented to the process to run as the root filesystem.
> 
> I think this also needs to be clarified. 

It's good to clarify the docs, but see [1,2] for previous discussion
in this PR.

[1]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r61261776
[2]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r62715247